src/Actions.js

```diff
@@ -0,0 +1,24 @@
+import ActionTypes from './ActionTypes';
+import Dispatcher from './Dispatcher';
+
+export function bootstrapStores(...stores) {
+  let payload = stores.map(store => ({name: store.name, state: store.dehydrate()}));
+  return {
+    type: ActionTypes.BOOTSTRAP_STORES,
+    payload
+  };
+}
```

Comment:
I think this will be solved by converting socket io service to fire flux actions on events. That way we can remove `BOOTSTRAP_STORES` action and bootstrap store instead during `CLIENT_CONNECTED` action.